docs/customization/subplots.ipynb

```diff
@@ -0,0 +1,148 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "32fb725d-602c-433b-8522-bd8be63fa4d0",
+   "metadata": {},
+   "source": [
+    "# Subplots / Tiled plots\n",
+    "\n",
+    "There are two ways to create subplot layouts with Plopp:\n",
+    "\n",
+    "- Using manually created Matplotlib axes\n",
+    "- Placing figures inside container widgets\n",
+    "\n",
+    "The first method will work with any (static or interactive) backend,\n",
+    "while the second will only work with the interactive (`%matplotlib widget`) backend.\n",
+    "\n",
+    "## Using Matplotlib axes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "40c4c95a-0b3c-4603-be82-f1555bba07bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import plopp as pp\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "da = pp.data.data_array(ndim=3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3b377e93-f7a5-4903-9bb2-a90c1844c3c7",
+   "metadata": {},
+   "source": [
+    "To create our subplots, we use Matplotlib's in-built `plt.subplots` function.\n",
+    "We then pass each subplot axes to Plopp's `plot` function as the `ax` argument:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "46d49d86-b72b-4083-98ff-6c322c1badca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots(2, 2, figsize=(8, 6))\n",
+    "\n",
+    "p1 = pp.plot(da['zz', 0], ax=ax[0, 0])\n",
+    "p2 = pp.plot(da['zz', 0]['yy', 0], ax=ax[0, 1])\n",
```

Comment:
Not related to this PR, but it is a bit odd that you have `pp.data.data_array` create data arrays that look like meant for testing (with 'xx', 'yy', ... as dim names).

Reply:
yeah, good point, I should probably change the names for the docs, would make more sense.

Reply:
I changed the labels to `x`, `y`, `z` by default, and set them to `xx`, `yy`, `zz` for the unit tests.